patches/chrome-common-channel_info_posix.cc.patch

```diff
@@ -0,0 +1,30 @@
+diff --git a/chrome/common/channel_info_posix.cc b/chrome/common/channel_info_posix.cc
+index d5dfbe13e273c204c34eea6bbae721e60a893496..6e37aa3a10388c4ed4691b13bd0d25ac668b6664 100644
+--- a/chrome/common/channel_info_posix.cc
++++ b/chrome/common/channel_info_posix.cc
+@@ -82,13 +82,24 @@ std::string GetDesktopName(base::Environment* env) {
+       return "google-chrome.desktop";
+   }
+ #else  // BUILDFLAG(CHROMIUM_BRANDING)
++#if defined(OFFICIAL_BUILD)
```

Comment:
why are we adding this? Also we should never have multiline patches like this anyway, it should be a `define`, but why aren't we just doing a chromium_src override here? I think maybe this was superceded by another commit that I commented on

Reply:
Yes, this should be removed since we completely override this file via chromium_src. Will remove.

Reply:
Removed in 14148ce6c7e3616560ae3c67c033de93a008893c